src/EpisodeIntegrationService/ReceiveData/ReceiveData.cs

```diff
@@ -154,6 +138,41 @@ private async Task ProcessEpisodeDataAsync(IEnumerable<BssEpisode> episodes, str
         }
     }
 
+    private const string DateFormat = "dd/MM/yyyy";
+    private EpisodeDto ProcessEpisode(BssEpisode episode)
+    {
+        return new EpisodeDto
+        {
+            EpisodeId = episode.episode_id,
+            EpisodeType = episode.episode_type,
+            NhsNumber = episode.nhs_number,
+            EpisodeOpenDate = string.IsNullOrEmpty(episode.episode_date) ? null : DateOnly.FromDateTime(DateTime.ParseExact(episode.episode_date, DateFormat, CultureInfo.InvariantCulture)),
+            AppointmentMadeFlag = GetAppointmentMadeFlag(episode.appointment_made),
+            FirstOfferedAppointmentDate = string.IsNullOrEmpty(episode.date_of_foa) ? null : DateOnly.FromDateTime(DateTime.ParseExact(episode.date_of_foa, DateFormat, CultureInfo.InvariantCulture)),
+            ActualScreeningDate = string.IsNullOrEmpty(episode.date_of_as) ? null : DateOnly.FromDateTime(DateTime.ParseExact(episode.date_of_as, DateFormat, CultureInfo.InvariantCulture)),
+            EarlyRecallDate = string.IsNullOrEmpty(episode.early_recall_date) ? null : DateOnly.FromDateTime(DateTime.ParseExact(episode.early_recall_date, DateFormat, CultureInfo.InvariantCulture)),
+            CallRecallStatusAuthorisedBy = episode.call_recall_status_authorised_by,
+            EndCode = episode.end_code,
+            EndCodeLastUpdated = string.IsNullOrEmpty(episode.end_code_last_updated) ? null : DateTime.ParseExact(episode.end_code_last_updated, "yyyy-MM-dd HH:mm:ssz", CultureInfo.InvariantCulture),
+            OrganisationCode = episode.bso_organisation_code,
+            BatchId = episode.bso_batch_id,
+            EndPoint = episode.end_point,
+            ReasonClosedCode = episode.reason_closed_code,
+            FinalActionCode = episode.final_action_code
+        };
+    }
+
+    private static string GetAppointmentMadeFlag(string appointmentMade)
```

Comment:
I've noticed we map this to a string and then later map it to a short. Let's update the Dto so that AppointmentMadeFlag is a short and then we can cut down on the amount of mapping